README.md

```diff
@@ -175,6 +181,50 @@ resource "aws_iam_role" "spacelift_scheduler_role" {
 
 The module is also available [on the OpenTofu registry](https://search.opentofu.org/module/spacelift-io/ecs-spacelift-selfhosted/aws/latest) where you can browse the input and output variables.
 
+## Additional Configuration Options
+
+### Message Broker Types
+
+This module supports two types of MQTT brokers:
+
+1. **Built-in MQTT broker (mqtt_broker_type = "builtin")**
+   - When using the built-in broker, the Spacelift server acts as the MQTT broker, but listens on a different port.
+     - A separate Network Load Balancer (NLB) is created for the MQTT broker port.
+     - The NLB doesn't require an ACM certificate. TLS is handled inside the Spacelift MQTT broker.  
+   - Requires DNS configuration (`CNAME`) for the MQTT broker endpoint.
+   - Set in your configuration with: `mqtt_broker_type = "builtin"` (this is the default), and `mqtt_broker_endpoint = "tls://<endpoint address>"`.
+
+2. **AWS IoT Core (mqtt_broker_type = "iotcore")**
```

Comment:
Maybe we should make it clear that if you use IoT Core, you *must* use SQS instead of the built-in message queue? IoT Core relies on the `iot` queue to allow the backend to receive IoT messages.

Reply:
true, I completely missed this

Reply:
updated